dcpy/connectors/postgres.py

```diff
@@ -3,6 +3,12 @@
 import csv
```

Comment:
I figure `psql` generally refers to postgres command line utility, this is a more accurate name

Reply:
Good call. I think I called named it `psql` before sqlalchemy crept it.